integreat_cms/static/src/js/feedback/toggle-feedback-entries.ts

```diff
@@ -0,0 +1,21 @@
+import { icons } from "lucide";
+import { toChildArray } from "preact";
+
+/*
+ * This file provides a simple spoiler for feedback list entries with hide/show functionality.
+ */
+window.addEventListener("icon-load", () => {
+    // Check which feedback entries need the toggle button
+    document.querySelectorAll(".feedback-entry-content").forEach((element) => {
+        if ((element as HTMLElement).offsetWidth >= element.scrollWidth) {
+            // Remove toggle button if not necessary
+            element.parentElement.querySelector(".toggle-feedback-entry").remove();
+        }
+    });
+    // Set event handlers for expanding/collapsing feedback entries
+    document.querySelectorAll(".toggle-feedback-entry").forEach((toggle) => {
+        toggle.addEventListener("click", ({ target }) => {
+            toggle.parentElement.classList.toggle("active");
+        });
+    });
+});
```

Comment:
Could you remove the old file [integreat_cms/static/src/js/feedback/toggle-feedback-comments.ts](https://github.com/digitalfabrik/integreat-cms/blob/4f6a66e7a63308faf4f1450383967bec730f7536/integreat_cms/static/src/js/feedback/toggle-feedback-comments.ts) since it has been replaced by this one?